src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -1071,9 +1073,10 @@ PeerImp::onMessage (std::shared_ptr <protocol::TMTransaction> const& m)
             std::weak_ptr<PeerImp> weak = shared_from_this();
             app_.getJobQueue ().addJob (
                 jtTRANSACTION, "recvTransaction->checkTransaction",
-                [weak, flags, stx] (Job&) {
+                [weak, flags, checkSignature, stx] (Job&) {
```

Comment:
Right.  That instructs SConstruct to use `-std=c++14` instead of `-std=c++11`.  And we need `-std=c++14` for this language feature (else we fail to compile).